packages/window-size-tool/rules.json

```diff
@@ -3,6 +3,9 @@
     "width": "< 1200",
     "terms": ["//CLUSTER/@NUM_COLS > 4", "count(//LIST[count(FIELD | CONTAINER) > 6 ]) > 0"
     ],
+    "andTerms": [
```

Comment:
What do you think of the property name andTerms ? Is it intuitive?

Together we have `terms` and `andTerms`.

What about renaming them to `anyTerms` and `allTerms`.

example docs would be

`anyTerms` - the rule will apply if any of these terms return true and the other criteria of the rule is met.
`allTerms` - the rule will only apply if all these terms return true and the other criteria of the rule is met.